features/netsocket/RAWSocket.cpp

```diff
@@ -0,0 +1,184 @@
+/*
+ * $ Copyright Cypress Semiconductor Apache2 $
```

Comment:
fix the license header per our license guide

Reply:
+ new feature like this needs to be in separate PR

Reply:
features/netsocket/RAWSocket.cpp addition is now tracked in https://github.com/ARMmbed/mbed-os/pull/10978

Reply:
@vmedcy thanks for these, will there be an update to this PR to remove the other functionality now tracked in separate PRs? I can't see one yet :)